[2023-01-04T12:19:14.691+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_pipeline.dashboard manual__2023-01-04T12:16:53.467375+00:00 [queued]>
[2023-01-04T12:19:14.704+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_pipeline.dashboard manual__2023-01-04T12:16:53.467375+00:00 [queued]>
[2023-01-04T12:19:14.704+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-04T12:19:14.705+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2023-01-04T12:19:14.705+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-04T12:19:14.724+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): dashboard> on 2023-01-04 12:16:53.467375+00:00
[2023-01-04T12:19:14.743+0000] {standard_task_runner.py:55} INFO - Started process 2939 to run task
[2023-01-04T12:19:14.749+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'accidents_pipeline', 'dashboard', 'manual__2023-01-04T12:16:53.467375+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/Airflow.py', '--cfg-path', '/tmp/tmp9r2t_jts']
[2023-01-04T12:19:14.752+0000] {standard_task_runner.py:83} INFO - Job 28: Subtask dashboard
[2023-01-04T12:19:14.833+0000] {task_command.py:389} INFO - Running <TaskInstance: accidents_pipeline.dashboard manual__2023-01-04T12:16:53.467375+00:00 [running]> on host 0eb9bfbdd4ca
[2023-01-04T12:19:14.920+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=accidents_pipeline
AIRFLOW_CTX_TASK_ID=dashboard
AIRFLOW_CTX_EXECUTION_DATE=2023-01-04T12:16:53.467375+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-04T12:16:53.467375+00:00
[2023-01-04T12:19:18.157+0000] {logging_mixin.py:137} INFO - Dash is running on http://0.0.0.0:8020/
[2023-01-04T12:19:18.156+0000] {dash.py:1968} INFO - Dash is running on http://0.0.0.0:8020/

[2023-01-04T12:19:18.163+0000] {logging_mixin.py:137} INFO -  * Serving Flask app '***'
[2023-01-04T12:19:18.164+0000] {logging_mixin.py:137} INFO -  * Debug mode: off
[2023-01-04T12:19:18.165+0000] {_internal.py:224} INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8020
 * Running on http://172.18.0.4:8020
[2023-01-04T12:19:18.166+0000] {_internal.py:224} INFO - [33mPress CTRL+C to quit[0m
[2023-01-04T12:19:23.638+0000] {_internal.py:224} INFO - 172.18.0.1 - - [04/Jan/2023 12:19:23] "GET / HTTP/1.1" 200 -
[2023-01-04T12:19:23.876+0000] {_internal.py:224} INFO - 172.18.0.1 - - [04/Jan/2023 12:19:23] "GET /_dash-dependencies HTTP/1.1" 200 -
[2023-01-04T12:19:23.884+0000] {_internal.py:224} INFO - 172.18.0.1 - - [04/Jan/2023 12:19:23] "GET /_favicon.ico?v=2.7.1 HTTP/1.1" 200 -
[2023-01-04T12:19:24.115+0000] {_internal.py:224} INFO - 172.18.0.1 - - [04/Jan/2023 12:19:24] "GET /_dash-layout HTTP/1.1" 200 -
[2023-01-04T12:19:24.188+0000] {_internal.py:224} INFO - 172.18.0.1 - - [04/Jan/2023 12:19:24] "[36mGET /_dash-component-suites/dash/dcc/async-graph.js HTTP/1.1[0m" 304 -
[2023-01-04T12:19:24.197+0000] {_internal.py:224} INFO - 172.18.0.1 - - [04/Jan/2023 12:19:24] "[36mGET /_dash-component-suites/dash/dcc/async-plotlyjs.js HTTP/1.1[0m" 304 -
[2023-01-04T12:23:37.848+0000] {local_task_job.py:224} WARNING - State of this instance has been externally set to success. Terminating instance.
[2023-01-04T12:23:37.851+0000] {process_utils.py:133} INFO - Sending Signals.SIGTERM to group 2939. PIDs of all processes in the group: [2939]
[2023-01-04T12:23:37.851+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 2939
[2023-01-04T12:23:37.852+0000] {taskinstance.py:1483} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-01-04T12:23:37.904+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=2939, status='terminated', exitcode=0, started='12:19:13') (2939) terminated with exit code 0
